etc/deploy/load.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+REGISTRY="${1}"
+
+docker load -o pachd
+docker tag
```

Comment:
I think you meant to remove this. This doesn't do anything, right?

Reply:
Same here, will remove.